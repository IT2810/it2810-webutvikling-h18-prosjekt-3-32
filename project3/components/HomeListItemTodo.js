import React from 'react';
import { TouchableOpacity, StyleSheet, Text, View } from 'react-native';
import styles from "../stylesheets/HomeListItem.style.js";


export default class HomeListItemTodo extends React.Component {

    render() {
        return (
            <React.Fragment>
                <View style={styles.itemContainer}>
                    <Text style={styles.itemText}>{this.props.name}</Text>
                    <Text style={{color:"#7f7f7f", fontSize:12}}>{this.props.date}</Text>
                </View>
            </React.Fragment>
        );
    }
};
