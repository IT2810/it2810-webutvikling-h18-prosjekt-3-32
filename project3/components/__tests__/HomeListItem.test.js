// Test that components mounts as expected

import React from 'react';
import HomeListItemCalendar from '../HomeListItemCalendar';
import HomeListItemTodo from '../HomeListItemTodo';
import renderer from 'react-test-renderer';


test('HomeListItemCalendar.js renders correctly?', () => {
    const stepTree = renderer.create(<HomeListItemCalendar/>);
    expect(stepTree).toMatchSnapshot();
})
