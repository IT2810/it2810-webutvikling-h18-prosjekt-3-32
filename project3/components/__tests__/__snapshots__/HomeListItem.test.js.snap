// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeListItemCalendar.js renders correctly? 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ededed",
      "borderTopWidth": 0.5,
      "height": 70,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#7f7f7f",
        "fontSize": 16,
        "marginBottom": 10,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#7f7f7f",
        "fontSize": 12,
      }
    }
  >
     - 
  </Text>
</View>
`;

exports[`HomeListItemCalendar.js renders correctly? 2`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ededed",
      "borderTopWidth": 0.5,
      "height": 70,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#7f7f7f",
        "fontSize": 16,
        "marginBottom": 10,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#7f7f7f",
        "fontSize": 12,
      }
    }
  />
</View>
`;

exports[`HomeListItemTodo.js renders correctly? 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ededed",
      "borderTopWidth": 0.5,
      "height": 70,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#7f7f7f",
        "fontSize": 16,
        "marginBottom": 10,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#7f7f7f",
        "fontSize": 12,
      }
    }
  />
</View>
`;
